grammar.js

```diff
@@ -8,19 +9,20 @@
      - either 'new file', 'modified', 'renamed' or 'deleted'
  */
 
+/**
+ * Whitespace that is ignored by the parser: tab, form feed, vertical tab, and
+ * space characters. Note that whitespace doesn't include newlines!
+ */
 const WHITE_SPACE = /[\t\f\v ]+/;
 const NEWLINE = /\r?\n/;
 const ANYTHING = /[^\n\r]+/;
-const CHANGE = choice("new file", "modified", "renamed", "deleted");
 const PREC = {
   NONSENSE: -1,
   PATH: 5,
   PATH_SEPARATOR_ARROW: 6,
-  REBASE_COMMAND: 7,
```

Comment:
There weren't any references to `PREC.REBASE_COMMAND` or `PREC.SUBJECT`, so I deleted them.